scripts/sysman/engine.yml

```diff
@@ -131,6 +147,24 @@ params:
       name: pStats
       desc: "[in,out] Will contain a snapshot of the engine group activity counters."
 --- #--------------------------------------------------------------------------
+type: function
+desc: "Get the activity stats for each Virtual Function (VF) associated with engine group. This function is used from a Physical Function (PF) interface when GPU is virtualized (SRIOV) into Virtual Function and Physical Function devices"
+class: $sEngine
+name: GetActivityExt
+details:
+    - "The application may call this function from simultaneous threads."
+    - "The implementation of this function should be lock-free."
+params:
+    - type: $s_engine_handle_t
+      name: hEngine
+      desc: "[in] Handle for the component."
+    - type: uint32_t*
+      name: pCount
+      desc: "[in,out] Pointer to the number of engine stats descriptors. The driver shall update the value with the total number of components of this type as provided in the count."
+    - type: $s_engine_stats_t*
+      name: pStats
+      desc: "[in,out] Will contain a snapshot of the engine group activity counters."
```

Comment:
Recommend writing `desc:` as:

```
      desc: |
            [in,out][optional][range(0, *pCount)] array of engine group activity counters.
              - if count is less than the total number of components available, the driver shall only retrieve that number of components.
```

The `pStats` must be an **optional** parameter so that the user may retrieve the count without validation flagging a `nullptr` parameter.

Reply:
Fixed